modules/textures/src/lib/parsers/parse-npy.ts

```diff
@@ -1,6 +1,17 @@
 // import type {TextureLevel} from '@loaders.gl/schema';
 
 type NumpyHeader = {descr: string; shape: number[]};
+type ArrayTypes =
```

Comment:
I suspect this type is availble in loader-utils or similar?

Reply:
The problem is that here I want to refer to the _constructor_ of various typed arrays. TS doesn't seem to allow `typeof` on a union of types:

```ts
import type {TypedArray} from '@loaders.gl/loader-utils/src/types';
type ConstructorType = typeof TypedArray;
// Error: 'TypedArray' only refers to a type, but is being used as a value here.ts(2693)
```

Reply:
There is a non-exported `TypedArrayConstructor` in gltf
https://github.com/visgl/loaders.gl/blob/fa3d2d0e464676b491f8581871058f9d3b8c79ec/modules/gltf/src/lib/gltf-utils/gltf-utils.ts#L5-L15

I'll rename this type here to be consistent